applications/empire_application/README.md

```diff
@@ -0,0 +1,8 @@
+# Overview
```

Comment:
Hi Guys,

this is a blocker. We can NOT link, and even less distribute a GPL library...

[EDIT]: this really worries me. @adityaghantasala @jcotela @pooyan-dadvand @rolandwuechner how can we solve this? Should we move the empire application to a dependent repository?
I am really not sure of what we should do, just that we are not  allowed to keep it here